vault-github/README.md

```diff
@@ -14,7 +15,7 @@ This module lets you authenticate with [Hashicorp Vault](https://www.vaultprojec
 ```tf
 module "vault" {
   source     = "registry.coder.com/modules/vault-github/coder"
-  version    = "1.0.2"
+  version    = "1.0.3"
```

Comment:
Shouldn't this be bumped separately from PR, wouldn't the `bun run update-version` overwrite this anyway?

Reply:
yes it will be overwritten :D